vim_onenote.ahk

```diff
@@ -117,77 +216,77 @@ Send, {End}{Enter}
 Gosub InsertMode
 return
 
+r::
+    send +{right}
+    gosub, InsertMode
+    ; Wait for single key to be pressed, sends that key and returns to normal
+    input, inp, V E L1
+    send {left}
+    gosub, NormalMode
+return
+
 ; undo
 u::Send, ^z
-return 
 ; redo.
 ^r::Send, ^y
-return 
 
-+G:: Send, ^{End}
 ; G goto to end of document
-return
+sG(){
```

Comment:
I think no, because while you gain readability, you lose functionality.
Autohotkey allows dynamic function calls (e.g. %var%() ). This method of naming the function is consistent, so that I can see that G is uppercase, and know that the function name will be "s"+"G"